ecr/Makefile

```diff
@@ -0,0 +1,31 @@
+all: init plan apply aws build tag push
```

Comment:
add a few variables here on the top:
ECR_REPO ?= ""
TAG ?= latest
IMAGE ?= $(ECR_REPO):$(TAG)

we can update those values in the future

Reply:
also included another variable IMAGE_NAME ?= "hello-world"